backend/pkg/commons/db2/parser.go

```diff
@@ -0,0 +1,195 @@
+package db2
+
+import (
+	"encoding/json"
+	"errors"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+
+	"github.com/gobitfly/beaconchain/pkg/commons/db2/jsonrpc"
+	"github.com/gobitfly/beaconchain/pkg/commons/db2/raw"
+)
+
+type GethTrace struct {
+	TxHash string
+	Result *GethTraceCall
+}
+
+type GethTraceCall struct {
+	TransactionPosition int
+	Time                string
+	GasUsed             string
+	From                common.Address
+	To                  common.Address
+	Value               string
+	Gas                 string
+	Input               string
+	Output              string
+	Error               string
+	RevertReason        string // todo have a look at this, it could improve revert message
```

Comment:
```suggestion
	RevertReason        string // todo have a look at this, it could improve revert message
```
Is this comment still needed or can we remove it?

Reply:
Yes it for future use. That field provide more context but is not present everywhere. If we change the indexer I want us to think about parsing this field.